app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def hello
+  	render html: "Hello world!"
```

Comment:
- indent = 2 khoảng trắng => 
```ruby
def hello
  render html: "Hello world!"
  #something
```
- Bạn tìm hiểu `I18n in ruby on rails` để sửa các lỗi về I18n